packages/nextjs-mf/src/internal.ts

```diff
@@ -93,40 +93,36 @@ export const DEFAULT_SHARE_SCOPE: SharedObject = {
 };
```

Comment:
The function 'isInternalOrPromise' can be made more generic by accepting an array of prefixes as a parameter. This will increase the modularity of the function, allowing it to be used in different contexts.